lesson-6/Alam - Lesson 6/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Mock Portfolio Site</title>
+</head>
+<body>
+
+  <nav>
+    <h3>My Portfolio Site</h3>
+    <ul>
+      <li><a href="#">My Work</a></li>
+      <li><a href="#">About Me</a></li>
+      <li><a href="#">Contact Me</a></li>
+    </ul>
+  </nav>
+
+  <header class="featured-project">
+    <div>
+      <h1>My Featured Project</h1>
+      <time>February 1, 2018</time>
+      <p>This is the coolest project I’ve ever done. It showcases many of my awesome skills. I can code and I can make stuff. Click on this button below to read more about my sweet project.</p>
+      <a href="#">Read More</a>
+    </div>
+    <img src="images/pic_placehold_lrg.png" />
+  </header>
+
+  <div class="set1">
+    <div class="first-pic">
+      <div>
+        <h2>Another Cool Project</h2>
+        <time>February 1, 2018</time>
+        <p>This is another project I’ve done that’s not as important as the big thing I want to show, but still cool.</p>
+        <a href="#">Read More</a>
+      </div>
+      <img src="images/pic_placehold_sm.png"
+    </div>
+
+
+    <div class="second-pic">
+      <div>
+        <h2>Another Cool Project</h2>
+        <time>February 1, 2018</time>
+        <p>This is another project I’ve done that’s not as important as the big thing I want to show, but still cool.</p>
+        <a href="#">Read More</a>
+      </div>
+      <img src="images/pic_placehold_sm.png"
+    </div>
+  </div>
+
+  <div class="set2">
+    <div class="third-pic">
+      <img src="images/pic_placehold_med.png"
+      <div>
+        <h2>Another Cool Project</h2>
+        <p>This is another piece of work I’ve done that I want to share, but it’s a smaller project.</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+
+    <div class="fourth-pic">
+      <img src="images/pic_placehold_med.png"
+      <div>
+        <h2>Another Cool Project</h2>
+        <p>This is another piece of work I’ve done that I want to share, but it’s a smaller project.</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+
+    <div class="fifth-pic">
+      <img src="images/pic_placehold_med.png"
+      <div>
+        <h2>Another Cool Project</h2>
+        <p>This is another piece of work I’ve done that I want to share, but it’s a smaller project.</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="set3">
+    <div class="sixth-pic">
+      <img src="images/pic_placehold_med-wide.png"
+      <div>
+        <h2>Another Cool Project</h2>
+        <p>This is another project I’ve done that’s not as important as the big thing I want to show, but still cool. This is another project I’ve done that’s not as important as the big thing I want toshow, but still cool.</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <nav>

```

Comment:
A better semantic HTML element for this part would be `<footer></footer>` (which we haven't learned specifically, so don't worry). Most often, a web page only has one `nav` element